test/test.sql

```diff
@@ -1,7 +1,7 @@
 DROP TABLE person_test;
```

Comment:
This was already here, I'm trying not to modify it. The errors are all ignored when run by pretest.js.